### ISSUE TITLE
NTR: Abnormalities of the glomerular endothelial cell (level 3)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6305]
Abnormalities of the glomerular endothelial cell (level 3)
parent:Abnormalities of the glomerular capillary lumina ()
def: @avizrosenberg @yongqunh

**Comment by pnrobinson** [comment]
Adding with label
*Abnormal glomerular endothelial cell morphology*
but as a child of https://hpo.jax.org/app/browse/term/HP:0025006 (Abnormal glomerular capillary morphology) -- this seems more appropriate than Abnormalities of the glomerular capillary lumina.
@avizrosenberg @yongqunh -- please reopen if this is wrong.
